scripts/catalog.rst

```diff
@@ -0,0 +1,336 @@
+Model Catalog
+=============
+
+
+
+
+Language Model
+--------------
+`Language Model Model Zoo Index <./language_model/index.html>`_
+
+Word Language Model
+~~~~~~~~~~~~~~~~~~~
+
+Dataset: Wikitext-2
+
++---------------------------------------+-----------------+-----------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------+
+| Pre-trained Model                     | Test Perplexity |Training Command                                                                                                             | log                                                                                                                         |
```

Comment:
Can we replace all `Training Command ` to `command`? For some tables, the first column is not wide enough.

Reply:
OK. I will make corresponding changes by next commit